src/components/layout/layout.js

```diff
@@ -131,7 +131,15 @@ const Layout = ({
     const [gtm_data, setGTMData] = useGTMData()
 
     const is_static = type === 'static'
-
+    const queryParams = new URLSearchParams(window.location.search)
+    const term = queryParams.get('platform')
```

Comment:
I think calling it `page` would be more descriptive than `term`.

Reply:
Ok , we can update. Thanks